client/app/src/main/java/com/example/criengine/Activities/RootActivity.java

```diff
@@ -97,10 +116,30 @@ public void onPageSelected(int position) {
             super.onPageSelected(position);
             int id = navigation.getMenu().getItem(position).getItemId();
             navigation.setSelectedItemId(id);
-            // TODO: Refresh stuff goes here.
+            // TODO: More refresh stuff goes here.
+            updateNotificationBadge();
         }
     }
 
+    /**
+     * Updates the Notification menu item badge to show the number of notifications
+     */
+    private void updateNotificationBadge() {
+        dbw.getProfile(dbw.userId).addOnSuccessListener(
+                new OnSuccessListener<Profile>() {
+                    @Override
+                    public void onSuccess(Profile profile) {
+                        int notificationCount = dummyProfile.getNotifications().size();
+                        BadgeDrawable badge = navigation
+                                .getOrCreateBadge(R.id.bottom_navigation_item_notifications);
+                        // TODO: uncomment the next line when notifications work

```

Comment:
What line must be uncommented? Nothing after this is commented lmao

Reply:
Oh potatoes, I forgot to remove this

Reply:
fixed